nixos-org/webserver.nix

```diff
@@ -154,6 +154,8 @@ in
                 #skins = [ ./wiki-skins ];
                 extraConfig =
                   ''
+                    $wgSiteNotice = "The NixOS wiki is outdated and is being shut down. If you wish to add or improve NixOS documentation, please grab a ticket from the [https://github.com/NixOS/nixpkgs/milestone/8 Move the wiki!] milestone.";
```

Comment:
s/ticket/issue/.

I think we should also say that each wiki page has an issue, so that people will know how to proceed if they want to add something based on the current page.